klangbecken.liq

```diff
@@ -110,24 +110,24 @@ def check_next_func(r) =
 end
 
 # Priority playlist for 'play next' feature
-prio = playlist.once(id="prio", reload_mode="watch", path.concat(DATA_DIR, "prio.m3u"))
```

Comment:
You should leave the `.once`, to double ensure that the prio playlist never loops.